common/membership/ringpop/factory_test.go

```diff
@@ -60,17 +61,20 @@ type (
 		internodeConfigMutualTLS config.GroupTLS
 		internodeConfigServerTLS config.GroupTLS
 
-		mutualTLSFactoryA *factory
-		mutualTLSFactoryB *factory
-		serverTLSFactoryA *factory
-		serverTLSFactoryB *factory
+		mutualTLSFactoryA     *factory
+		mutualTLSFactoryB     *factory
+		serverTLSFactoryA     *factory
+		serverTLSFactoryB     *factory
+		serverIPv6TLSFactoryA *factory
+		serverIPv6TLSFactoryB *factory
 
 		insecureFactory *factory
 	}
 )
 
 const (
 	localhostIPv4 = "127.0.0.1"
+	localhostIPv6 = "0::1"
```

Comment:
What's the extra zero indicate? I've always seen this as:
```suggestion
	localhostIPv6 = "::1"
```

Reply:
there are lots of ways to write it - the zeros (127 bits worth) are implied. but i've seen some IP parsers accept `0::1` and not `::1` for some reason. we can make this `::1` tho - should be fine either way.